backend/app/controllers/api/v1/admin/ddahs_controller.rb

```diff
@@ -115,18 +115,7 @@ def find_ddah
     end
 
     def ddah_params
-        params.slice(
-            :assignment_id,
-            :approved_date,
-            :accepted_date,
-            :revised_date,
-            :emailed_date,
-            :signature
```

Comment:
We want to only allow `assignment_id`, the rest is updated in the DdahService or thorugh individual `accept`, `approve`, and `email` routes.